### ISSUE TITLE
AppendChild gives unexpected behavior

### ISSUE THREAD

**Issue by fredrikholm** [zzzprojects/html-agility-pack#133]
TL;DR: I have a tbody node to which I want to add two rows, but instead of appending at the end, the added rows end up what the last being first in the collection, just like they were inserted at position zero.

Long story: I have a specification pre-populated with one tr that serves as a template (with some cells which in turn contains values). Here are the steps I take:
1. select the tbody node
2. select the pre-populated row
3. call RemoveAllChildren on the tbody node
4. loop over my dataset and clone the row found in (2), manipulate the cells and add it to the tbody using AppendChild

The rows now appear in reverse order.

Previous/Next siblings seems correct, and I've also tried to manipulate the line numbers, to no avail.

Maybe I've misunderstood the way it's supposed to work?

**Comment by fredrikholm** [comment]
Ooops - just ignore this, turned out I manipulated stuff in the wrong way. Sorry. :-)

**Comment by JonathanMagnan** [comment]
Hehe np ;) Thank you for letting me know.

Best Regards,

Jonathan
